ODIN_II/regression_test/benchmark/task/syntax/synthesis_result.json

```diff
@@ -2091,6 +2093,8 @@
         "verilog": "part_select.v",
         "warnings": [
             "part_select.v:15:1 [NETLIST] Net (null) driving node top.inst_3^bAND~0^lAND~7 is itself undriven.",
+            "part_select.v:15:1 [NETLIST] Net (null) driving node top.inst_3^bAND~0^lAND~7 is itself undriven.",
```

Comment:
How come this is duplicated?

Reply:
print_dot_names_header() check this condition twice now, once in warn_undriven() and once in print_input_single_driver(). I can probably remove the first call without any issue though if it matters?

Reply:
Yeah, I don't think it's necessary to keep both. Given undriven will be either dropped or printed out, you should get the warning eventually